pilot/pkg/networking/util/util.go

```diff
@@ -44,7 +44,15 @@ const (
 	PassthroughCluster = "PassthroughCluster"
 	// SniClusterFilter is the name of the sni_cluster envoy filter
 	SniClusterFilter = "envoy.filters.network.sni_cluster"
-
+	// IstioMetadataKey is the key under which metadata is added to a route or cluster
+	// regarding the virtual service or destination rule used for each
+	IstioMetadataKey = "networking.istio.io"
```

Comment:
For EDS, this is just "istio". I think we can use "istio" here too (less space).